frontend/src/components/landing-page/landing-page.tsx

```diff
@@ -0,0 +1,86 @@
+import { Link } from 'react-router-dom';
+
+import Car from '@assets/images/landing-page/picture.png';
+import { AppRoute } from '@common/enums/app/app-route.enum';
+import { CarsCategories } from '@components/cars-categories/cars-categories';
+import { ButtonFill } from '@components/common/button-fill/button-fill';
+import { ButtonOutline } from '@components/common/button-outline/button-outline';
+import { NewCarCard } from '@components/new-car-card/new-car-card';
+import { TopCarCard } from '@components/top-car-card/top-car-card';
+import { Container } from '@mui/material';
+
+import { Header } from './components/components';
+
+import './styles.scss';
+
+export const LandingPage = (): React.ReactElement => {
+  return (
+    <>
+      <Header />
+      <Container
+        sx={{ maxWidth: '100% !important' }}
+        className="main-container"
+      >
+        <div className="first-container">
+          <div className="first-container-left">
+            <div className="first-container-left-text-main">
```

Comment:
Don't use `className` with strings! Use the exported classes from the SCSS. 
`className={styles.someClass}`